### PR TITLE
Update local dev setup instructions

### PR DESCRIPTION
**Pull request by dusvyat**
## Description

setup.py is deprecated - https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

## Type of change

(Please delete options that are not relevant)

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] 📄 This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [ ] Relevant unit and integration tests are updated or added.

### CODE REVIEW

**Comment by dusvyat** [comment]
**Don't merge until this is merged**: https://github.com/mindsdb/mindsdb/pull/8265

**Comment by martyna-mindsdb** [comment]
Thanks @dusvyat 

This PR is going to have conflicts with [this PR](https://github.com/mindsdb/mindsdb/pull/8399). Once it is merged, I'll add this alternative way using makefile.

**Review by martyna-mindsdb — APPROVED** [review]

**Pull request merged**
